packages/story-editor/src/elements/media/imageDisplay.js

```diff
@@ -108,6 +108,8 @@ function ImageDisplay({ element, box, previewMode }) {
           // Ignore
         }
       });
+    } else {
+      setSrc(resource.src);
     }
```

Comment:
Previous code was only settings the `src` when the full-size image was not loaded (so it was preloading the image and then settings the `src`) and did nothing when we already had full-size in resourseList (and component was updating), now, if we have the full-size image we just set the `src`.

Reply:
> it's there because we first see the preview of it when dropping?

✅  Yea this is the behavior I was seeing when I was logging it out yesterday